community/contributing/docs_writing_guidelines.rst

```diff
@@ -277,25 +277,43 @@ How to write methods and classes
 Dynamic vs Static typing
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Despite the addition of optional static typing, GDScript remains primarily 
-dynamic language. Unless you are specifily discussing static typing, you
-should only use dynamic typing in your code examples and snipets. 
-This will help reduce confusion to new users by promoting consistent look
-to the code examples they are exposed to. 
+GDScript aims to be concise and easy to learn. Optional static typing conflicts with this vision
+by making documentation more congested. To improve its accessibility for new users, please use 
+dynamic GDScript code samples where possible. The one exception is topics that explain 
+static typing concepts to users.
 
-**Don't** write using using static typing:
+**Don't** add a type hint with a colon or by casting:
 
 ::
 
     var a := 5
     var b: String = "My String"
+    var my_sprite : Sprite = $Sprite as Sprite
```

Comment:
Idiomatic static typing would be to use a `:=` for type inference since the cast implies what the type will be inferred as. No need to restate the "Sprite" class twice.

    var my_sprite := $Sprite as Sprite

Reply:
By the way, between writing the type and using casting, which would you favor? `var sprite : Sprite = get_node("Body")` or `var sprite := get_node("Body") as Sprite`?